### ISSUE TITLE
pynac.pc is not complete when --with-giac has been selected

### ISSUE THREAD

**Issue by thierry-FreeBSD** [pynac/pynac#352]
If pynac is built against Giac, -lgiac is missing from the produced pkg-config file pynac.pc, and linkage fails.

Such a patch should be applied in this case:

```
--- pynac.pc.in.orig    2019-05-13 07:15:21.000000000 +0200
+++ pynac.pc.in 2020-04-21 18:39:30.147227000 +0200
@@ -7,6 +7,6 @@
 Description: C++/Python library for symbolic calculations
 Version: @VERSION@
 Requires: python-@PYTHON_VERSION@ factory
-Libs: -L${libdir} -lpynac
+Libs: -L${libdir} -lpynac -lgiac
 Libs.private: @LIBGIAC@ -lflint -lgmp
 Cflags: -I${includedir}
```

Note: a real patch should be given for configure.ac, but I have not enough autotools knowledge for that, it's why I don't send a pull request, sorry.

**Comment by dimpase** [comment]
You mean to say that `@LIBGIAC@` should be moved from `Libs.private:` to `Libs`, right? Could it be something FreeBSD-specific (I vaguely recall some differences in regard the indirect deps in linking)?

**Comment by thierry-FreeBSD** [comment]
Yes, I had to add -lgiac to Libs, Libs.private is not sufficient.
But this occurs only with libgiac: -lflint -lgmp belongs to Libs.private and everything is fine with them.
Maybe this a specific problem with FreeBSD (the port math/giacxcas is a bit strange) and in this case, we can close this issue: the port of pynac has a work-around.

**Comment by dimpase** [comment]
Could it be due to libgiac being C++ rather than C library - different linking model, dunno? Moving of libgiac to Libs.private happened on #343 - not because it broke anything, but just due to "standards". 

Can we see a log of what happens with libgiac in  Libs.private on FreeBSD?

**Comment by dimpase** [comment]
fixed by 35429db
